osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -94,7 +98,10 @@ private void load()
                 HitObjectArea,
                 new SkinnableDrawable(new ManiaSkinComponent(ManiaSkinComponents.KeyArea), _ => new DefaultKeyArea())
                 {
-                    RelativeSizeAxes = Axes.Both
+                    RelativeSizeAxes = Axes.Both,
+                    // This is pretty dodgy (and will cause weirdness when pausing gameplay) but is better than completely broken rewind.
+                    Clock = host.UpdateThread.Clock,
+                    ProcessCustomClock = false,
```

Comment:
I don't get it. The inline comment really does no job at explaining the actual issue this is solving, and it doesn't seem to break anything by removing this.

What am I missing?

Reply:
I thought the complete brokenness was obvious enough (repros ~100% for me):

https://user-images.githubusercontent.com/191335/194539483-f865fa66-e361-469e-afa0-04e273423dff.mp4

To explain why it fixes: It allows the key area to exist in forwards-only time, and not worry about rewind at all. Which is much easier than attempting to get them to look correct during rewind.

Why it breaks during pause: `FrameStabilityContainer` pauses all update flow, so they *will* correctly stop animating, but on resuming they will immediately update to their final state (as the host clock has still been elapsing during the time paused).

Reply:
I didn't notice it because it doesn't seem to be majorly broken in the triangles ruleset. Basically this is only broken because these particular elements have a lighting element in classic/argon.

Reply:
Added better documentation, see how that reads to you.